react/features/welcome/components/WelcomePageLists.js

```diff
@@ -74,15 +80,35 @@ class WelcomePageLists extends Component<Props> {
      * @inheritdoc
      */
     render() {
-        const { disabled } = this.props;
+        const { disabled, _hasCalendarEntries } = this.props;
 
         return (
             <PagedList
-                defaultPage = { 0 }
+                defaultPage = { _hasCalendarEntries ? 1 : 0 }
                 disabled = { disabled }
                 pages = { this.pages } />
         );
     }
 }
 
-export default translate(WelcomePageLists);
+/**
+ * Maps (parts of) the redux state to the React {@code Component} props of
+ * {@code WelcomePageLists}.
+ *
+ * @param {Object} state - The redux state.
+ * @protected
+ * @returns {{
+ *     _hasCalendarEntries: boolean
+ * }}
+ */
+function _mapStateToProps(state: Object) {
+    const { events } = state['features/calendar-sync'];
+
+    console.log('ZB:_mapStateToProps', events);
+
+    return {
+        _hasCalendarEntries: events && events.length
```

Comment:
maybe wrap in a `Boolean(...)` just to make sure we always pass a boolean and not undefined some times